src/rta.rs

```diff
@@ -36,7 +36,7 @@ impl<'a> ValidationReport<'a> {
         &self,
         engine: &Engine,
     ) -> Result<(), Failed> {
-        let mut run = engine.start(self)?;
+        let mut run = engine.start(self, true)?;
```

Comment:
Must `update` indeed always be `true` for RTAs?

Reply:
Not sure. But since RTA is dead, I don’t want to spend too much time on it.